src/js/post-data.js

```diff
@@ -0,0 +1,18 @@
+require('@babel/polyfill');
+
+const id = '982X2Sxledkcq4ce1LDh';
+const URL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores/`;
+
+const postData = async (user, score) => {
+  await fetch(URL, {
+    method: 'POST',
+    body: JSON.stringify({ user, score }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => data);
+};
```

Comment:
**Optional Changes**
- [ ] Nice try using the `async await` syntactic sugar to wrap your asynchronous code. However, you're mixing it with the `then` keyword. Let us remove the `then` keyword and just use the pure `async await`.

Reply:
Yeah sure. Let me remove those then to make it pure async await. Thanks for pointing out the mistake.